pyproject.toml

```diff
@@ -49,6 +49,8 @@ toml = {version=">=0.10.2", optional=true}
 tzlocal = "^5.0"
 python-decouple = ">=3.8"
 click = ">=8.0.0"
+nptdms = "^1.9.0"
+matplotlib = {version=">=3.9.0", python = ">=3.9"}
```

Comment:
Oh, I see. Yeah, we shouldn't add dependencies here for examples. Instead just let the example document what a user should install.

Reply:
You can add a `[tool.poetry.group.example.dependencies]` group for 3rd party dependencies that are only needed by example programs. This will install these dependencies when you run `poetry install` but not when you run install `pip install nidaqmx` or `poetry install --only main`.

Reply:
BTW the difference is that `[tool.poetry.dependencies]` defines what packages `nidaqmx` depends on, whereas `[tool.poetry.group.*.dependencies]` defines additional dependency groups that are only used when developing nidaqmx-python.

Reply:
Correction: plural `examples` makes more sense when writing `poetry install --with examples` etc.